src/blueprints/apikeys.py

```
@@ -0,0 +1,131 @@
+            # Reload environment variables
+            for key, value in valid_entries:
+                os.environ[key] = value
+            
+            return jsonify({
+                "success": True,
+                "message": f"Saved {len(valid_entries)} API key(s). Some plugins may require restart to pick up changes."
+            })
+        else:
+            return jsonify({"error": "Failed to write .env file"}), 500
+            
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)